### DISCUSSION TITLE
how I can reset changelog

### DISCUSSION THREAD

**Discussion by Frank17489** [netbox-community/netbox#16440] [General]
Hi,

I create many changelog entries via a migration procedure. Is it possible to clear this after I finish the migration?

thx for help

**Comment by a084ed22** [comment]
You have multiple options depending on the behavior you desire.
configuration.py has a parameter, CHANGELOG_RETENTION, after which the internal housekeeping jobs will purge the changelog. Alternatively, you'll have to connect to the database and manually remove the entries you don't want.

**Comment by Frank17489** [comment]
Ok, thanks for the information. is table **extras_objectchange** the right one?

**Reply by a084ed22** [reply]
Yes, that is the table where changelogs are stored.

**Comment by Frank17489** [comment]
it works, thx
